Packages/install_dependencies

```diff
@@ -18,12 +18,7 @@ python-pip \
 python-setuptools \
 python-dev \
 python-virtualenv \
-swig \
-socat \
 libyaml-dev \
-python-scipy \
-python-numpy \
```

Comment:
Numpy needs to remain as a dist-installed module.

Reply:
It's still there in install_redeem.

Reply:
It's OK, I was thinking that Toggle needed it also.